nginx/tests/test_nginx.py

```diff
@@ -33,7 +33,8 @@ def test_connect_ssl(check, instance_ssl, aggregator):
     instance_ssl['ssl_validation'] = False
     check_no_ssl = check(instance_ssl)
     check_no_ssl.check(instance_ssl)
-    aggregator.assert_metric("nginx.net.connections", tags=TAGS, count=1)
+    extra_tags = ['nginx_host:{}'.format(HOST), 'port:{}'.format(PORT_SSL)]
```

Comment:
This is non-blocking: I think I would define a constant or a fixture for `extra_tags` and pass that around. In `test_plus_api.py` I would do the same for `base_tags`. LMK if that's clear.